### DISCUSSION TITLE
Adaptive icons

### DISCUSSION THREAD

**Discussion by davecates** [airsdk/Adobe-Runtime-Support#3027] [General]
I can't find any documentation on how to utlise adaptive icons with an Air app. I've seen some comment that they're now supported with the Air SDK but can't find out how to use them.

Can anyone help?

Thanks.

**Comment by ajwfrost** [comment]
I think you can do this via custom resources... there's mention on how to package up custom resources here:
https://airsdk.dev/docs/tutorials/platform/android/custom-resources

Although there's also a tutorial specifically about Android Adaptive Icons available from Distriqt:
https://docs.airnativeextensions.com/docs/tutorials/android-adaptive-icons/

thanks

**Comment by davecates** [comment]
Fantastic! That second link from Distriqt was perfect and what I was looking for. All sorted.

Thanks for the very prompt reply as well! :)

**Comment by amorganiv** [comment]
Adding a note that Android Studio now exports the mipmap image assets in the WebP format. Would it be possible for AIR to support packaging WebP images?

**Reply by marchbold** [reply]
There shouldn't be any limitation currently for AIR to support webp images as android resources. However if you are looking to load them into your app you may need to use a decoder (eg https://airnativeextensions.com/extension/com.distriqt.WebP)

**Reply by amorganiv** [reply]
OK... but in my testing AIR failed to package the WebP images. It only worked after converting them to PNG images.

**Reply by marchbold** [reply]
Oh that's interesting, you mean as resources in the `resdir`?

**Reply by amorganiv** [reply]
Correct! The mipmap folder assets.

**Reply by marchbold** [reply]
Interesting, I'll try to setup a test and replicate.
